@here/harp-mapview/lib/MapView.ts

```diff
@@ -1752,6 +1755,18 @@ export class MapView extends THREE.EventDispatcher {
         return this.m_fog;
     }
 
+    set elevationWrapperThingy(elevationWrapperThingy: ElevationWrapperThingy | undefined) {
```

Comment:
I wonder if we even need the ElevationWrapperThingy. We could also just have multiple parameters.

Reply:
Good point, would be one less class.

Reply:
Removed.